package/rules

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@ --with autoreconf --with systemd --parallel
+
+# These are used for cross-compiling and for saving the configure script
+# from having to guess our platform (since we know it already)
+DEB_HOST_GNU_TYPE   ?= $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
+DEB_BUILD_GNU_TYPE  ?= $(shell dpkg-architecture -qDEB_BUILD_GNU_TYPE)
+CROSS= --build $(DEB_BUILD_GNU_TYPE) --host $(DEB_HOST_GNU_TYPE)
+
+# Build-deps not available on these archs
+DEB_BUILD_ARCH ?= $(shell dpkg-architecture -qDEB_BUILD_ARCH)
+ifeq (,$(filter $(DEB_BUILD_ARCH), hurd-i386 kfreebsd-amd64 kfreebsd-i386))
+    UNBOUND = --with-unbound
+else
+    UNBOUND =
+endif
+
+ifeq (,$(filter $(DEB_BUILD_ARCH), hurd-i386))
+    ODBX = --with-odbx
+else
+    ODBX =
+endif
+
+
+override_dh_auto_configure:
+	dh_testdir
+	CONFIG_SHELL=/bin/sh  ./configure $(CROSS) --prefix=/usr --mandir=\$${prefix}/share/man --infodir=\$${prefix}/share/info \
+		--sysconfdir=/etc \
+		CFLAGS="$(CFLAGS) $(CPPFLAGS)" \
+		LDFLAGS="$(LDFLAGS) -Wl,-z,defs -L/usr/lib/libmilter" \
+		--disable-live-testing \
+		--enable-vbr \
+		--enable-rbl \
+		--enable-atps \
+		--enable-stats \
+		--enable-replace_rules \
+		--enable-query_cache \
+		--with-libmemcached \
+		--with-openldap \
+		--with-db \
+		--with-sql-backend \
+		--with-sasl \
+		--with-test-socket=inet:8891@localhost \
+		--with-domain=localhost \
+		--with-lua \
+		--enable-poll \
+		--enable-sender_macro \
+		--with-tre \
```

Comment:
added --with-tre and --enable-diffheaders for dkim_diffheaders feature support

Reply:
You can remove this rules file, I've merged in the new Docker image that compiles these features into `libopendkim`.